assets/src/components/ZoomLevelWrapper.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactElement, useState } from "react"
+import { useMap } from "react-leaflet"
+
+const ZoomLevelWrapper = ({
+  render,
+}: {
+  render: (zoomLevel: number) => ReactElement
+}) => {
```

Comment:
nit(non-blocking): I'd personally like the `render` the prop to be named `children` instead, but I'm more curious as to if there was a style or personal reason for using `render` over any other name.